src/pages/requests/_evidence-types.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
We've ended up with these files back where they were, when they're already in `test/fixtures`